extensions/interactions/Continue/validator.js

```diff
@@ -26,18 +26,18 @@ oppia.filter('oppiaInteractiveContinueValidator', [
 
     baseInteractionValidationService.requireCustomizationArguments(
       customizationArgs, ['buttonText']);
-    
+
     if (customizationArgs.buttonText.value.length === 0) {
       warningsList.push({
         type: WARNING_TYPES.CRITICAL,
-        message: 'the button text should not be empty.'
+        message: 'The button text should not be empty.'
```

Comment:
Not particularly. It was something that was irritating me (the warnings showing up in the graph kept starting with a lowercase letter) and I figured (perhaps wrongly) that I might as well fix it since doing so wasn't hard. In retrospect it would probably have been better to leave it to a separate PR.